### PR TITLE
fix: device in use error modal

### PR DESCRIPTION
**Pull request by eswarclynn**
### Details(context, link the issue, how was the bug fixed, what does the new feature do)

-
-

### CODE REVIEW

File: packages/roomkit-react/src/Prebuilt/components/Notifications/DeviceInUseModal.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    >
+      <DialogContent title="Device Access Error">
+        <DialogRow>
+          <Text variant="body2">
+            We weren't able to access your camera since it's either in use by another application or is not configured
+            properly. Please follow the following instructions to resolve this issue.
+          </Text>
+        </DialogRow>
+        <Instruction
+          number="1"
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/Notifications/DeviceInUseModal.tsx:35]: can be an unorder list or order list. no need to pass numbers

File: packages/roomkit-react/src/Prebuilt/components/Notifications/Notifications.tsx [hunk 2]
```diff
@@ -98,6 +100,21 @@ export function Notifications() {
           }
           return;
         }
+        if (notification.data?.code === 3003) {
```
Comment (raviteja83) [inline packages/roomkit-react/src/Prebuilt/components/Notifications/Notifications.tsx:103]: let us move this and the modal to a new component like permission error modal

File: packages/roomkit-react/src/Prebuilt/components/Notifications/DeviceInUseModal.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    <Dialog.Root
+      open={showDeviceInUseModal}
+      onOpenChange={() => {
+        setShowDeviceInUseModal(false);
+      }}
+    >
+      <DialogContent title="Device Access Error">
+        <DialogRow>
+          <Text variant="body2">
+            We weren't able to access your camera since it's either in use by another application or is not configured
```
[the remark this answers is not in the document]
Comment (eswarclynn) [reply packages/roomkit-react/src/Prebuilt/components/Notifications/DeviceInUseModal.tsx:30]: Sure will add. Just checking, the actual device in use by another application issue is only with camera in windows right?

[the remark this answers is not in the document]
Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/Notifications/DeviceInUseModal.tsx:30]: happens with both audio and video.

[the remark this answers is not in the document]
Comment (raviteja83) [reply packages/roomkit-react/src/Prebuilt/components/Notifications/DeviceInUseModal.tsx:30]: majorly video but audio as well.

**Review by raviteja83 — DISMISSED** [review]

**Review by KaustubhKumar05 — APPROVED** [review]

**Review by raviteja83 — APPROVED** [review]

**Review by raviteja83 — APPROVED** [review]

**Review by KaustubhKumar05 — APPROVED** [review]

**Pull request merged**